flipper/events/types.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 eShares, Inc. dba Carta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import enum
+
+
+class EventType(enum.Enum):
+    PRE_CREATE = "PRE_CREATE"
```

Comment:
Why not just make these lowercase so that you don't have to convert in the event emitter for the subscriber calls?